ovos_utils/network_utils.py

```diff
@@ -1,6 +1,27 @@
 import socket
+
 import requests
 
+from ovos_utils.log import LOG
+
+
+def get_network_tests_config():
```

Comment:
This could be made internal if its only intended to be used by the methods in this module, though it shouldn't make a difference with the way Configuration is initialized